src/scss/components/_thumbnail.scss

```diff
@@ -16,21 +16,29 @@
 **/
 
 .thumbnail {
-  width: 100%;
   height: 100%;
   position: relative;
   overflow: hidden;
-  border-radius: $borderRadius;
-  transition: transform .2s linear;
+  border-radius: $borderRadiusButton;
   will-change: transform;
+  width: 98%;
+  max-height: 190px;
+  margin: 0 auto;
+  box-shadow: 0 -2px 4px 0 get-color((color: blackNeutral, opacity: .1));
+  transition: transform .2s ease-in-out;
 
   &--action {
     display: block;
     transform: scale(-1);
 
     &:hover {
-      transition: transform .2s linear;
-      transform: translateY(-7px) scale(-1.03) translateZ(1px);
+      transform: translate(0, -5px) scale(-1);
```

Comment:
@eduardojmatos scale(-1) is to keep the thumbnail from being upside down on hover. We already use it with no side effects.